mitmproxy/addons/script.py

```diff
@@ -121,8 +128,7 @@ async def watcher(self):
                 mtime = os.stat(self.fullpath).st_mtime
             except FileNotFoundError:
                 ctx.log.info("Removing script %s" % self.path)
-                scripts = list(ctx.options.scripts)
-                scripts.remove(self.path)
+                scripts = get_scripts()
                 ctx.options.update(scripts=scripts)
```

Comment:
This updates the option to the explicit values, is that intentional?

Reply:
Yes, I thought it was okay to do so.